Source/BSN.IpTables.Api/Program.cs

```diff
@@ -22,6 +22,7 @@ public static void Main(string[] args)
 
             startup.Configure(app, app.Environment);
 
+            Console.WriteLine("IpTablesProgram:Main(): before run");

```

Comment:
Why use `Console.WriteLine`? please use `ILogger` instead of this.

Reply:
That was a debug log, worthless for production. Got removed.